notebooks/114-quantization-simplified-mode/114-quantization-simplified-mode.ipynb

```diff
@@ -0,0 +1,346 @@
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!mo --framework=onnx --data_type=FP32 --input_shape=[1,3,32,32] -m $onnx_model_path  --output_dir $MODEL_DIR"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Compression stage\n",
+    "Compress the model with the following command:\n",
+    "  \n",
+    "`pot -q default -m <path_to_xml> -w <path_to_bin> --engine simplified --data-source <path_to_data>`"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!pot -q default -m $ir_model_xml -w $ir_model_bin --engine simplified --data-source $CALIB_DIR --output-dir compressed --direct-dump --name $MODEL_NAME"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Compare Performance of the Original and Quantized Models\n",
+    "\n",
+    "Finally, we will measure the inference performance of the FP32 and INT8 models. To do this, we use [Benchmark Tool](https://docs.openvino.ai/latest/openvino_inference_engine_tools_benchmark_tool_README.html) - an inference performance measurement tool for OpenVINO.\n",
+    "\n",
+    "**NOTE:** For more accurate performance, we recommended running benchmark_app in a terminal/command prompt after closing other applications. Run benchmark_app -m model.xml -d CPU to benchmark async inference on CPU for one minute. Change CPU to GPU to benchmark on GPU. Run benchmark_app --help to see an overview of all command line options."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "optimized_model_path = Path('compressed/optimized')\n",
+    "optimized_model_xml = optimized_model_path / '{}.xml'.format(MODEL_NAME)\n",
+    "optimized_model_bin = optimized_model_path / '{}.bin'.format(MODEL_NAME)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Inference FP32 model (IR)\n",
+    "!benchmark_app -m $ir_model_xml -d CPU -api async"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Inference INT8 model (IR)\n",
+    "!benchmark_app -m $optimized_model_xml -d CPU -api async"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Demonstration of the results\n",
+    "\n",
+    "This section demonstrates how to use the compressed model by running the optimized model on a subset of images from the CIFAR10 dataset and shows predictions using the model.\n",
+    "\n",
+    "The first step is to load the model:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "ie = Core()\n",
+    "\n",
+    "compiled_model = ie.compile_model(str(optimized_model_xml), \"AUTO\")"
+   ]
```

Comment:
@Noktyrn After running this, I got this error.

![image](https://user-images.githubusercontent.com/10940214/161106591-e9a4b378-230e-45b9-99fb-8c655db02290.png)

Reply:
Sorry @paularamo, but I couldn't reproduce this issue. Can you please tell me, if the IR was created in the compressed\optimized folder?